sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubAsyncClient.java

```diff
@@ -32,9 +34,13 @@
     private final boolean isSharedConnection;
     private final Runnable onClientClose;
     private final TracerProvider tracerProvider;
+    private final boolean enableIdempotentPartitions;
+    private final Map<String, PartitionPublishingState> initialPartitionPublishingStates;
 
-    EventHubAsyncClient(EventHubConnectionProcessor connectionProcessor, TracerProvider tracerProvider,
-        MessageSerializer messageSerializer, Scheduler scheduler, boolean isSharedConnection, Runnable onClientClose) {
+    EventHubAsyncClient(
+        EventHubConnectionProcessor connectionProcessor, TracerProvider tracerProvider,
```

Comment:
I'm not sure why the arguments are moved to the next line.

Reply:
Updated.